experimental/auth-logic/wrappers/endorsement_wrapper_test.go

```diff
@@ -29,21 +33,16 @@ func TestEndorsementWrapper(t *testing.T) {
 
 }`
 
-	testEndorsementWrapper := endorsementWrapper{
-		endorsementFilePath: testEndorsementPath,
+	testEndorsementWrapper := EndorsementWrapper{
+		EndorsementFilePath: testEndorsementPath,
 	}
 
-	endorsement, err := ParseEndorsementFile(testEndorsementPath)
-	if err != nil {
-		t.Fatalf("couldn't parse endorsement file: %s error: %v",
-			testEndorsementPath, err)
-	}
-	validEndorsement, err := endorsement.GenerateValidatedEndorsement()
-	if err != nil {
-		t.Fatalf("couldn't validate endorsement struct: %v error: %v",
-			endorsement, err)
-	}
-	speaker := fmt.Sprintf(`"%s::EndorsementFile"`, validEndorsement.Name)
+  endorsementAppName, err := GetAppNameFromEndorsement(testEndorsementPath)
+  if err != nil {
+    t.Fatalf("couldn't get name from endorsement file: %s, error: %v",
+      testEndorsementPath, err)
+  }
+	speaker := fmt.Sprintf(`"%s::EndorsementFile"`, endorsementAppName)
```

Comment:
nit: The indentation does not look quite right.

Reply:
should be fixed by next push